_test/check_exercises.nim

```diff
@@ -63,9 +63,10 @@ Options:
   -t, --tmp       Set output path to a temporary location (e.g. in `/tmp`)"""
   quit(0)
 
-let
-  appDir = getAppDir()
-  exercisesDir = appDir / ".." / "exercises"
+const
+  thisDir = currentSourcePath().parentDir()
```

Comment:
It may be more obvious to refer to the root dir of the repo instead of `thisDir` which can be confusing 🤷‍♂️

Reply:
Thanks - I agree that `thisDir` is a bad name.

Pushed 8c4f73c84284 - I'd like to keep the default output dir as `_test`, at least in this PR.